keylime/cloud_verifier_tornado.py

```diff
@@ -433,9 +438,9 @@ def process_agent(self, agent, new_operational_state):
                         cloud_verifier_common.notifyError(agent,'comm_error')
                     else:
                         logger.debug("Communication error for new agent.  no notification will be sent")
-                    self.process_agent(agent, cloud_verifier_common.CloudAgent_Operational_State.FAILED)
+                    await self.process_agent(agent, cloud_verifier_common.CloudAgent_Operational_State.FAILED)
                 else:
-                    cb = functools.partial(self.invoke_get_quote, agent, True)
+                    cb = functools.partial(await self.invoke_get_quote, agent, True)
```

Comment:
I think this might be the issue, since we're `await`ing a function that's being passed in (and not executed).